src/SCREENS.ts

```diff
@@ -265,7 +265,6 @@ const SCREENS = {
     },
 
     EDIT_REQUEST: {
-        ROOT: 'EditRequest_Root',
```

Comment:
Why is this removed in this PR?

Reply:
EditRequestPage uses underscore but the page itself isn't being used anymore, so I remove it too.

Reply:
Can you clarify how come its not used yet? what other pages are used to edit request?

Reply:
I searched for the usage of the edit request route and component and found nothing.

It's previously used when we are editing money request fields, but we have combined the edit and create page into one like in this [PR](https://github.com/Expensify/App/pull/39723).